objc/objc_block_darwin_test.go

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 The Ebitengine Authors
+
+package objc_test
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+	"unsafe"
+
+	"github.com/ebitengine/purego"
+	"github.com/ebitengine/purego/objc"
+)
+
+func ExampleNewBlock() {
+	_, err := purego.Dlopen("/System/Library/Frameworks/Foundation.framework/Foundation", purego.RTLD_GLOBAL)
```

Comment:
Although it isn't necessary it's good to add `purego.RTLD_NOW|` as that's the proper way to call dlopen

Reply:
Okay, will fix. I actually copied the syntax from the existing load of  `libobjc.dylib`.

Reply:
Please fix it there too